modules/rosa-delete-dedicated-admins.adoc

```diff
@@ -6,7 +6,7 @@
 
 :_content-type: PROCEDURE
 [id="rosa-delete-dedicated-admins_{context}"]
-= Revoking `dedicated-admin` access using the `rosa` CLI
+= Revoking `dedicated-admin` access using the {product-title} (ROSA) CLI (`rosa`)
```

Comment:
Title too long, probably just CLI here

Reply:
As I mentioned earlier, maybe "ROSA CLI". Let me know if you have other thoughts.